### ISSUE TITLE
Facebook integration

### ISSUE THREAD

**Issue by aquaskualos** [pH7Software/pH7-Social-Dating-CMS#626]
**Describe the bug**
Hi.
I am trying to configure the app to activate registration through Facebook but it always gives me the same error, no matter how much I try and try. Nor have I found information on the network.
The error message I receive is
URL blocked
The redirection failed because the URI is not included in the white list of the app's OAuth client configuration. Verify that OAuth client access and OAuth web access are enabled, and add all domains of the app as valid OAuth redirect URIs.
I think everything is well configured.
Has it happened more times?

**To Reproduce**
1. Go to 'xcitame.com'
2. Click on 'universal login'
3. cilick on  'facebook'

**Expected behavior**
, must enter ph7 account or register.

**Comment by polynamaude** [comment]
You need to add your site url to the authorized domain for callback on the application configuration page for your Facebook App.

**Comment by aquaskualos** [comment]
Thanks for answering

I've tried it anyway. I currently have valid OAuth redirect URIs

https://xcitame.com/_protected/library/Service/Microsoft/Live/oauth_client.php
https://xcitame.com/connect/main/login/fb
https://xcitame.com/
https://www.xcitame.com/
https://www.xcitame.com/_protected/library/Service/Microsoft/Live/oauth_client.php
https://www.xcitame.com/connect/main/login/fb
https://www.facebook.com/v2.7/dialog/
https://www.xcitame.com/fb-callback.php
https://xcitame.com/fb-callback.php

If only the domain name does not work either.
I have tried it in a thousand ways.



De: Polyna-Maude R.-Summerside [mailto:notifications@github.com]
Enviado el: domingo, 01 de septiembre de 2019 13:29
Para: pH7Software/pH7-Social-Dating-CMS
CC: aquaskualos; Author
Asunto: Re: [pH7Software/pH7-Social-Dating-CMS] Facebook integration (#626)


You need to add your site url to the authorized domain for callback on the application configuration page for your Facebook App.

**Comment by aquaskualos** [comment]
Yes, yes, I understand, where should I put the URL to authorize?
I think I already have everything right.

**Comment by aquaskualos** [comment]
![fbdvlprs](https://user-images.githubusercontent.com/18622417/64133327-06153e00-cdde-11e9-9416-57d248c201ad.jpg)

**Comment by polynamaude** [comment]
I don't speak spanish

**Comment by aquaskualos** [comment]
Oh I'm sorry. But can you tell if the urls are correct?
You said I had to add the authorized domain, but I think this is already done.

**Comment by aquaskualos** [comment]
Hi.
I asked a question after looking for information, so as not to disturb.
If I have uploaded the images it is because the answer given has not been the solution to the query.
Even after explaining that I did not understand what I should do, I have been given the exact same answer, and after that I have uploaded an image of my configuration in the Facebook app.

I use the translator to make the inquiries because my English is horrible.

Can someone confirm if the settings I have on the facebook app are correct?
